seed-config.yml

```diff
@@ -0,0 +1,6 @@
+autoproj_test_utility_default: true
+autoproj_test_utility:
+  base/types: false
+  orogen: false
```

Comment:
Are these disabled because they take too long to run?

Reply:
They would take too long to run, and take a long time to install. Since they are not the point of the build-test setup, they are disabled here.